types/pem/index.d.ts

```diff
@@ -135,6 +135,12 @@ export interface CertificateSubjectReadResult {
     emailAddress: string;
 }
 
+export interface Pkcs12ReadResult{
```

Comment:
**Lint error:** Missing whitespace between name and bracket

Reply:
thanks for pointing out this, added missing whitespace